code/examples/channel/src/state.rs

```diff
@@ -81,6 +82,7 @@ impl State {
     pub async fn received_proposal_part(
         &mut self,
         from: PeerId,
+        from_pub_key: &PublicKey,
```

Comment:
Let's remove this and instead pass the validator set (accessible in `app.rs` via `genesis.initial_validator_set`

```suggestion
        validator_set: &ValidatorSet,
```

Reply:
Perfect. Is there a particular reason why the genesis is not included as part of the state?

Reply:
I will do that also then! Thanks!